beetsplug/lastgenre/__init__.py

```diff
@@ -179,12 +198,19 @@ def _resolve_genres(self, tags):
                     parents = [find_parents(tag, self.c14n_branches)[-1]]
 
                 tags_all += parents
-                if len(tags_all) >= count:
+                # Stop if we have enough tags already, unless we need to find
+                # the most specific tag (instead of the most popular).
+                if (not self.config['prefer_specific'] and
+                    len(tags_all) >= count):
```

Comment:
I think Travis's flake8 checker is unhappy about the alignment here: https://travis-ci.org/beetbox/beets/jobs/162943469

I'm honestly a little bewildered by this message, and a little angry at the tool for being so confusing, but I think PEP-8 might want you to indent the line one more set of 4 spaces, according to this issue: https://github.com/PyCQA/pycodestyle/issues/386

This error is weird, though—in the future, I might look into disabling it.